src/Headers/GhostNeighbours.hpp

```diff
@@ -527,14 +531,16 @@ struct DomainCellProxy
 : public TreeCellBase<ndim>
 {
 	using TreeCellBase<ndim>::hbox ;
-	using TreeCellBase<ndim>::rcell ;
+	//using TreeCellBase<ndim>::rcell ;
+  FLOAT rcell[ndim];
 
 	DomainCellProxy(const DomainBox<ndim>& box)
 	{
 		for (int k=0; k<ndim;k++)
 		{
 			rcell[k] = 0.5 * (box.max[k] + box.min[k]) ;
 			FLOAT dr = 0.6 * (box.max[k] - box.min[k]) ;
+      // DAVID : Why is this 0.6 and not 0.5 (or 0.5000001)??
```

Comment:
This object is no longer used, it was from the old brute force implementation. I'll remove it.